### PR TITLE
Issue 1 user can see the landing page

### PR DESCRIPTION
**Pull request by KiraHike**
image

### CODE REVIEW

File: css/layout.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  width: 9rem;
+  height: 11rem;
+}
+
+#zip-landing {
+  margin-top: 3rem;
+  width: 6rem;
+  padding: 0.25rem;
+  text-align: center;
+}
```
Comment (codingcodymiller) [inline css/layout.css]: These should be in `styles.css`, since they are not layout related

File: css/layout.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  margin: 0;
+  padding: 1rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+li.nav {
+  list-style: none;
+  display: inline;
+}
```
Comment (codingcodymiller) [inline css/layout.css]: These should be in `styles.css`, since they are not layout related

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css/layout.css (hunk 1 above)
Comment (KiraHike) [reply css/layout.css]: Got it.

File: css/layout.css (hunk 2 above)
Comment (KiraHike) [reply css/layout.css]: Got it.

**Comment by KiraHike** [comment]
image

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**